enterprise/server/util/ocicache/ocicache.go

```diff
@@ -23,6 +24,10 @@ import (
 	bspb "google.golang.org/genproto/googleapis/bytestream"
 )
 
+var (
+	cacheSecret = flag.String("oci.cache.secret", "", "Secret to add to OCI image cache keys.")
```

Comment:
you can add the `flag.Secret` tag to hide the value in certain places flag values are displayed.

Reply:
Nice, thanks!